components/symbl_ai/actions/put-member/put-member.mjs

```diff
@@ -0,0 +1,48 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+
+export default {
+  key: "symbl_ai-put-member",
+  name: "Update Member",
+  description: "Update an existing Conversation Member object. See the doc [here](https://docs.symbl.ai/docs/conversation-api/update-members/).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    conversationId: {
+      propDefinition: [
+        symblAIApp,
+        "conversationId",
+      ],
+    },
+    memberId: {
```

Comment:
Can we use [This method](https://docs.symbl.ai/docs/conversation-api/members/) to list all the members using `async options`? If so, it would be ideal to have this property on `app`

Reply:
@vellames-turing Would you have an example of a prop that depends on another prop to retrieve its values? The case for this one is that the `memberId` list can only be retrieved with a `conversationId` already defined. While I can retrieve the `conversationId` value, I will also need to use the `conversationId` to retrieve the list of members. Any guidance here would be great.

Reply:
Yeah, please take a look [here](https://github.com/PipedreamHQ/pipedream/blob/master/components/spotify/actions/remove-items-from-playlist/remove-items-from-playlist.mjs). Please let me know if you have more questions :)

Reply:
Brilliant... Thanks for the example. That's addressed in 733910c0.